src/getSearchKeywords.js

```diff
@@ -0,0 +1,53 @@
+function getSearchKeywords() {
+  const currentUrl = new URL(window.location.href);
+  const urlParams = new URLSearchParams(currentUrl.search);
+  const allCondition = ["+", " AND ", "|", " OR ", " -", `"`, " ", ","];
+  let keywords;
```

Comment:
맥락상 기본 태세로 변수 초기값을 지정하는 것은 어떨까요?
```suggestion
  const keywords = [];
```
혹여라도 의도가 있으시다면, undefined 보다는 null은 어떠신가요?
```suggestion
  let keywords = null;
```

Reply:
감사합니다. 의견 반영하여 수정하겠습니다~